modules/lsp/src/test/scala/playground/lsp/BuildLoaderTests.scala

```diff
@@ -0,0 +1,46 @@
+package playground.lsp
+
+import cats.effect.IO
+import fs2.io.file.Path
+import playground.BuildConfig
+import playground.language.TextDocumentProvider
+import weaver._
+
+object BuildLoaderTests extends SimpleIOSuite {
+  // won't actually be used, but still TODO: use a stub/fake
+  implicit val tdp: TextDocumentProvider[IO] = null
+
+  val loader = BuildLoader.instance[IO]
+
+  test("build loader can see stdlib") {
+    val result = loader
+      .buildSchemaIndex(BuildLoader.Loaded(BuildConfig(), Path(".")))
+      .map(_.allServices)
+
+    result.map { services =>
+      assert.eql(services.map(_.service.id.name), List("Random", "Clock"))
+    }
+  }
+
+  test("build loader can see external services") {
+    val result = loader
+      .buildSchemaIndex(
+        BuildLoader.Loaded(
+          BuildConfig(mavenDependencies =
+            Some(
+              List(
+                // todo
```

Comment:
this needs to happen - need to make sure this keeps working when some paths / deps are actually provided.

I'll probably need to publish a service locally for this, but I'll also see if AWS expose any.